index.js

```diff
@@ -762,6 +762,14 @@
     typeEq('sanctuary-def/Type')
   );
 
+  var VariadicType = NullaryType(
```

Comment:
Let's add a type signature along these lines:

```javascript
//  VariadicType :: Type | Type -> Type | Type -> Type -> Type | ...
```

The made-up nature of the signature will draw attention to our naughtiness.